flash/image/instance_segmentation/data.py

```diff
@@ -74,15 +177,24 @@ def train_default_transforms(self) -> Optional[Dict[str, Callable]]:
 
 
 class InstanceSegmentationOutputTransform(OutputTransform):
-    @staticmethod
-    def uncollate(batch: Any) -> Any:
-        return batch[DataKeys.PREDS]
+    # @staticmethod
+    # def uncollate(batch: Any) -> Any:
+    #     breakpoint()
+    #     return batch[DataKeys.PREDS]
+    def per_sample_transform(self, sample: Any) -> Any:
+        # TODO GET HERE WITH image SIZE
+        breakpoint()
+        resize = K.geometry.Resize(sample[DataKeys.METADATA]["original_size"][-2:], interpolation="bilinear")
+        sample[DataKeys.PREDS]["mask_array"] = (
+            resize(torch.tensor(sample[DataKeys.PREDS]["mask_array"], dtype=torch.float)) > 0
+        ).to(torch.uint8)
```

Comment:
The mask can have multiple classes so the `> 0` won't have the desired effect here as it casts everything to binary. I think a better way to do it would be with nearest. neighbour interpolation then just cast to a long tensor.